Update README.md (#9378)
gravitational/teleport: README.md

|Project Links| Description
|---|----
| [Teleport Website](https://goteleport.com/) | The official website of the project. |
| [Documentation](https://goteleport.com/docs/) | Admin guide, user manual and more. |
| [Demo Video](https://www.youtube.com/watch?v=b1WHFW0NIoM) | 3-minute video overview of Teleport. |
| [Blog](https://goteleport.com/blog/) | Our blog where we publish Teleport news. |
| [Forum](https://github.com/gravitational/teleport/discussions) | Ask us a setup question, post your tutorial, feedback, or idea on our forum. |
| [Slack](https://goteleport.com/slack) | Need help with your setup? Ping us in our Slack channel. |

> | [Cloud-hosted](https://goteleport.com/pricing) | We offer Teleport Pro and Enterprise with a Cloud-hosted option. For teams that require easy and secure access to their computing environments. |

Reviewer: Just confirming that we're still including messaging around Teleport Pro

> | [Cloud-hosted](https://goteleport.com/pricing) | We offer Teleport Pro and Enterprise with a Cloud-hosted option. For teams that require easy and secure access to their computing environments. |

Author: ```suggestion
| [Cloud-hosted](https://goteleport.com/pricing) | We offer Enterprise with a Cloud-hosted option. For teams that require easy and secure access to their computing environments. |
```

* Join [Teleport Discussions](https://github.com/gravitational/teleport/discussions) to ask questions. Our engineers are available there to help you.
* If you want to contribute to Teleport or file a bug report/issue, you can create an issue here in Github.
* If you are interested in Teleport Enterprise or more responsive support during a POC, we can also create a dedicated Slack channel for you during your POC. You can [reach out to us through our website](https://goteleport.com/pricing/) to arrange for a POC.

## Is Teleport Secure and Production Ready?

Teleport is used by leading companies to enable engineers to quickly access any

> computing resource anywhere. Teleport has completed several security audits from the nationally recognized technology security companies. [Some](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/) of

Reviewer: - Maybe remove "the" from "the nationally recognized technology security companies," unless we did get audits from all of them?
- Rather than the double link on "Some of them," maybe add a short bulleted list of public audits we've received, including the names of the orgs, to make them more prominent?

Author: I'm goign to add a link to https://goteleport.com/resources/audits/ 
```suggestion
computing resource anywhere. Teleport has completed several security audits from the nationally recognized technology security companies. We make some our audits public, view our latest [audit reports.](https://goteleport.com/resources/audits/).
```

> [them](https://goteleport.com/resources/audits/teleport-security-audit-2020/) have been made public. We are comfortable with the use of Teleport from a security perspective.

Author: ```suggestion
We are comfortable with the use of Teleport from a security perspective.
```
